CONTRIBUTING.md

```diff
@@ -214,8 +214,12 @@ When a person is assigned to an issue this indicates an _intent to implement_.
 
 * Include screenshots and animated GIFs in your pull request whenever possible.
 * List any steps necessary to trigger the feature you've created or bug you are fixing
-* Always run the [tests](#tests) locally before creating your PR
+* Always run the [unit tests](#unit-tests) locally before creating your PR
+ * You are not required to run the [integration tests](#integration-tests) locally
```

Comment:
Something like this might be better: "Run the unit tests locally before creating your PR to catch an issues early. You can run the integration tests too, but if you want to avoid setting up that environment, you can open a PR to run them automatically on CI."

You don't have to use my wording at all, but just making it clear that you can us the CI instead of a local setup to run the integration tests.